### PR TITLE
refactor: shortened orjson checks

### PR DESCRIPTION
**Pull request by Sengolda**
## Summary
the orjson checks in `utils.py` have been shortened (very sorry for typo in commit message)
## Information

- [] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed).
- [ ] This PR is **not** a code change (e.g. documentation, README, typehinting,
      examples, ...).

## Checklist

- [x] I have searched the open pull requests for duplicates.
- [x] If code changes were made then they have been tested.
  - [ ] I have updated the documentation to reflect the changes.
- [x] If `type: ignore` comments were used, a comment is also left explaining why.
- [ ] I have updated the changelog to include these changes.

### CODE REVIEW

File: discord/utils.py [hunk 1]
```diff
@@ -660,19 +660,12 @@ def _bytes_to_base64_data(data: bytes) -> str:
...
 
-    _from_json = orjson.loads  # type: ignore
 
-else:
-
-    def _to_json(obj: Any) -> str:
-        return json.dumps(obj, separators=(",", ":"), ensure_ascii=True)
-
-    _from_json = json.loads
+_from_json = orjson.loads if HAS_ORJSON else json.loads  # type: ignore
```
Comment (Lulalaby) [inline discord/utils.py:668]:
```suggestion
_from_json = orjson.loads if HAS_ORJSON else json.loads # type: ignore
```

**Comment by Lulalaby** [comment]
@Sengolda your broken pr spammed my phone notifications.

Please write the indentation correctly next time 

facepalm

**Comment by plun1331** [comment]
> @Sengolda your broken pr spammed my phone notifications.
> 
> Please write the indentation correctly next time
> 
> facepalm

sounds like a you problem

**Review by Lulalaby — APPROVED** [review]

**Comment by Om1609** [comment]
Checking `HAS_ORJSON` at define-time will be more efficient than checking it on every call to `_to_json`.

**Comment by Dorukyum** [comment]
I agree with @Om1609 on this one.
> Checking `HAS_ORJSON` at define-time will be more efficient than checking it on every call to `_to_json`.

**Pull request closed without merging**